testsuite/basic/pom.xml

```diff
@@ -29,13 +29,6 @@
     <dependency>
       <groupId>io.smallrye</groupId>
       <artifactId>smallrye-fault-tolerance</artifactId>
-      <exclusions>
-        <!-- https://github.com/eclipse-ee4j/interceptor-api/issues/31 -->
-        <exclusion>
-          <groupId>jakarta.ejb</groupId>
-          <artifactId>jakarta.ejb-api</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
This is now excluded in the implementation artifact, so no longer need to be excluded in test suite.